mod/cp_notifications/start.php

```diff
@@ -1532,8 +1532,8 @@ function cp_digest_daily_cron_handler($hook, $entity_type, $return_value, $param
 
 
 			// delete and clean up the notification, already sent so we don't need to keep it anymore
-			$query = "DELETE FROM notification_digest WHERE user_guid = {$user->getGUID()}";
-			$result = delete_data($query);
+			//$query = "DELETE FROM notification_digest WHERE user_guid = {$user->getGUID()}";
```

Comment:
Delete me?

Reply:
I commented that out so I can do testing (i had forgotten to uncomment when I did the commit)

Reply:
Np, you can use this https://help.github.com/articles/changing-a-commit-message/ to just fixup the last commit. You need to do a `git push -f` after though so make sure the push settings are correct (or you're using protected branches in the repo for safety)